defi/README.md

```diff
@@ -10,11 +10,13 @@ coverY: 0
 
 Ocean Protocol's open-source tools are useful in a variety of DeFi applications. Here we show you how to capitalize with Ocean Protocol tech in several ways using crypto trading!
 
-## Predicting the Price of ETH 📈
+## Data Farming - Challenge DF 📈
 
-We offer a Predict ETH data challenge where participants can win prizes 💰 for making accurate ETH price predictions. Participants use machine learning to extrapolate ETH prices for the next 12 hours as accurately as possible and upload these predictions to the Ocean Market as data NFTs. It's a win-win for both Ocean Protocol and challenge participants! This challenge promotes more valuable data feeds on the Ocean Market and participants gain cash prizes!
+Data Farming offers a weekly rewards via [Challenge DF](../../rewards/df-challengedf.md), a challenge where users predict the price of Ethereum and win prizes 💰 based on accuracy.
```

Comment:
The link towards the df-challengedf.md is broken. it should be ../rewards/df-challengedf.md

Reply:
fixed